Scenes/Outline.shader

```diff
@@ -0,0 +1,56 @@
+shader_type spatial;
+render_mode unshaded;
+
+uniform float scale = 2.0;
+uniform float threshold = 2.0;
+uniform vec3 color = vec3(0, 0, 0);
+uniform bool enabled = true;
+
+varying mat4 CAMERA;
```

Comment:
This is used?

Reply:
I'll comment out

Reply:
In the future or for enhancements if want to work in world space instead of view space then will need it.